doc/whats_new/v1.1.rst

```diff
@@ -303,6 +303,13 @@ Changelog
   :pr:`22300` by :user:`Meekail Zain <micky774>` and :pr:`15948` by
   :user:`sysuresh`.
 
+- |Enhancement| :class:`decomposition.FastICA` now allows the user to select
+  how whitening is performed through the new `svd_solver` parameter, which
+  supports `svd` and `eigh`. `svd_solver` defaults to `svd` although `eigh` may
+  be faster in cases where `num_features > num_samples` :pr:`11860` by
+  :user:`Pierre Ablin <pierreablin>`, :pr:`22527` by
+  :user:`Meekail Zain <micky774>` and :user:`Thomas Fan`_.
```

Comment:
We have custom links for maintainers (defined [here](https://github.com/scikit-learn/scikit-learn/blob/main/doc/whats_new/_contributors.rst))

```suggestion
  :user:`Meekail Zain <micky774>` and `Thomas Fan`_.
```